buildtest/utils/file.py

```diff
@@ -119,3 +119,24 @@ def create_dir(dirname):
         except OSError as err:
             print(err)
             raise
+
+
+def resolve_path(path):
+    """This method will resolve a file path with accounts for shell expansion and resolve paths in
```

Comment:
This method will resolve a file path _to account for_ shell expansion...

Reply:
fixed locally